html/Mogo/index.html

```diff
@@ -0,0 +1,1084 @@
+      <div class="banner__single-bar">
+        <span>03</span> About
+      </div>
+      <div class="banner__single-bar">
+        <span>04</span> Contact
+      </div>
+
+    </div>
+  </section>
+
+  <!-- About us -->
+  <section class="section about">
+    
+    <!-- Type -->
+    <p class="section-type section-type--black about__type">
+      What we do
+    </p>
+    
+    <!-- Title -->
+    <h2 class="section-title section-title--black about__title">
+      Story about us
+    </h2>
+
+    <!-- Description -->
+    <p class="section-description about__description">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
+    </p>
+
+    <!-- Images container -->
+    <div class="about__container">
+
+      <!-- Background -->
+      <div class="moving-image__bg">
+
+        <div class="moving-image__box">
+
+          <!-- Picture -->
+          <picture class="moving-image__picture">
+            <source media="(min-width : 1024px)" srcset="./assests/images/about/layer-44@3x.jpg">
+            <img src="./assests/images/about/layer-44@2x.jpg" alt="">
+          </picture>
+  
+          <!-- Overlay text -->
+          <div class="moving-image__overlay-content">
+            <img src="./assests/images/about/users.svg" alt="Computer">
+            <p class="small-title">Super team</p>
+          </div>
+        </div>
+
+      </div>
+
+
+      <!-- Background -->
+      <div class="moving-image__bg">
+
+        <div class="moving-image__box">
+
+          <!-- Picture -->
+          <picture class="moving-image__picture">
+            <source media="(min-width : 1024px)" srcset="./assests/images/about/layer-39@3x.jpg">
+            <img src="./assests/images/about/layer-39@2x.jpg" alt="Workplace">
+          </picture>
+  
+          <!-- Overlay text -->
+          <div class="moving-image__overlay-content">
+            <img src="./assests/images/about/users.svg" alt="">
+            <p class="small-title">Super team</p>
+          </div>
+        </div>
+
+      </div>
+
+
+      <!-- Background -->
+      <div class="moving-image__bg">
+
+        <div class="moving-image__box">
+
+           <!-- Picture -->
+          <picture class="moving-image__picture">
+            <source media="(min-width : 1024px)" srcset="./assests/images/about/layer-41@3x.jpg">
+            <img src="./assests/images/about/layer-41@2x.jpg" alt="Workplace">
+          </picture>
+  
+          <!-- Overlay text -->
+          <div class="moving-image__overlay-content">
+            <img src="./assests/images/about/users.svg" alt="">
+            <p class="small-title">Super team</p>
+          </div>
+        </div>
+
+      </div>
+
+
+     
+
+      
+    </div>
+
+  </section>
+
+  <!-- Counters -->
+  <section class="counters">
+    <div class="counters__container">
```

Comment:
I would say that you have counter here and after counter__number/counter__details